pkg/mountutil/mountutil_linux.go

```diff
@@ -120,6 +123,18 @@ func parseVolumeOptionsWithMountInfo(vType, src, optsRaw string, getMountInfoFun
 	var opts []string
 	var specOpts []oci.SpecOpts
 
+	if len(bindOpts) > 1 {
```

Comment:
I think we should check `if len(bindOpts) > 0 && vType != Bind` before  `len(bindOpts) > 1`
user should know from begining that neither `bind` nor `rbind` can be supported

Reply:
SGTM